### PR TITLE
Don't install nginx twice on OSX Travis build.

### PR DESCRIPTION
**Pull request by cesarbs**
cc @Tratcher @BrennanConroy

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,14 +6,9 @@ addons:
     packages:
     - libunwind8
 before_install:
-  - if test "$TRAVIS_OS_NAME" == "osx"; then brew update; brew install nginx; brew install openssl; brew link --force openssl; fi
+  - if test "$TRAVIS_OS_NAME" == "osx"; then brew update; brew install nginx; brew install openssl; brew link --force openssl; else ./install-nginx.sh; fi
```
Comment (BrennanConroy) [inline .travis.yml]: sh $PWD/install-nginx.sh ?

Comment (cesarbs) [inline .travis.yml]: What's the difference?

Comment (BrennanConroy) [inline .travis.yml]: Well `./install-nginx.sh` is failing to run

Comment (cesarbs) [inline .travis.yml]: There was a typo :grin:

**Comment by cesarbs** [comment]
Green. Squirrel?

**Comment by BrennanConroy** [comment]
Ship :ship:

**Pull request merged**